src/pages/mobile.tsx

```diff
@@ -0,0 +1,376 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Lottie from 'lottie-react';
+import { step1Lottie, step2Lottie, step3Lottie } from 'public/assets/lottie';
+import { COLOR } from 'src/styles/color';
+import { FONT_STYLES } from 'src/styles/fontStyle';
+import ImageDiv from '@src/components/common/ImageDiv';
+import { icMobileLogoWhite, icMobileLogo } from 'public/assets/icons';
+import { imgMobileBgFirst, imgMobileBgLast, imgMobileFist, imgMobileEar, imgMobileMic } from 'public/assets/images';
+
+function Mobile() {
+  const [isFirstScrolled, setIsFirstScrolled] = useState<boolean>(false);
+
+  const scrollListener = () => {
+    setIsFirstScrolled(window.scrollY > 0);
+  };
+  useEffect(() => {
+    window.addEventListener('scroll', scrollListener);
+    return () => {
+      window.removeEventListener('scroll', scrollListener);
+    };
+  }, []);
+
+  return (
+    <StMoblie>
+      <StNav isFirstScrolled={isFirstScrolled}>
+        <ImageDiv src={isFirstScrolled ? icMobileLogo : icMobileLogoWhite} className="logo" layout="fill" alt="" />
+      </StNav>
+
+      <StFirstSlide>
+        <p>
+          아나운서 쉐도잉으로 키우는 <br />
+          스피치 자신감, 딜리버블
+        </p>
+        <p>딜리버블은 PC에서 이용해 주세요.</p>
+      </StFirstSlide>
+
+      <StSecondSlide>
+        <p>
+          지금보다 더 잘 말하고
+          <br />
+          싶었던 적 없나요?
+        </p>
+        <div>
+          <StCard>
+            <ImageDiv src={imgMobileEar} className="ear" layout="fill" alt="" />
+            <div>
+              <p>
+                발음을 교정하고 싶은데, <br />
+                학원 가격이 너무 부담돼요.
+              </p>
+              <p>26살 취준생 김○○</p>
+            </div>
+          </StCard>
+          <StCard>
+            <ImageDiv src={imgMobileFist} className="fist" layout="fill" alt="" />
+            <div>
+              <p>
+                목소리가 떨려서 발표를 망쳤어요.
+                <br />
+                저도 당당하게 말하고 싶어요!
+              </p>
+              <p>23살 대학생 류○○</p>
+            </div>
+          </StCard>
+          <StCard>
+            <ImageDiv src={imgMobileMic} className="mic" layout="fill" alt="" />
+            <div>
+              <p>
+                아나운서 준비를 시작했는데, <br />
+                뉴스리딩은 처음이라 막막해요.
+              </p>
+              <p>26살 취준생 김○○</p>
```

Comment:
```suggestion
              <p>24살 아나운서 지망생 백○○</p>
```

![image](https://user-images.githubusercontent.com/58380158/209670449-4ce7eaa6-126a-406c-9e03-fba7e37211fd.png)

요거 반영 부탁해용!

Reply:
아하 넵